set_test.go

```
@@ -46,6 +47,71 @@
+}
+
+func BenchmarkSet_Clear(b *testing.B) {
+	b.ReportAllocs()
+	var memStats runtime.MemStats
+
+	sets := make([]Set[int], 0)
+
+	runtime.ReadMemStats(&memStats)
+	beforeAlloc := memStats.Alloc
+
+	for b.Loop() {
```

golangci-lint (1.64.8)

stdversion: testing.Loop requires go1.24 or later (file is go1.22)

(govet)